src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.cs

```diff
@@ -619,6 +628,44 @@ private BoundExpression MakeImplicitConversion(BoundExpression rewrittenOperand,
                 rewrittenType: rewrittenType);
         }
 
+        private BoundExpression RewriteTupleConversion(
+            CSharpSyntaxNode syntax,
+            BoundExpression rewrittenOperand,
+            ConversionKind conversionKind,
+            bool @checked,
+            bool explicitCastInCode,
+            TupleTypeSymbol rewrittenType)
+        {
+            var destElementTypes = rewrittenType.TupleElementTypes;
+            var numElements = destElementTypes.Length;
+
+            var srcType = (TupleTypeSymbol)rewrittenOperand.Type;
+            var srcEleemntTypes = srcType.TupleElementTypes;
```

Comment:
I think the array is there, ready to use.

---
In reply to: [65587124](https://github.com/dotnet/roslyn/pull/11681#discussion_r65587124) [](ancestors = 65587124)